frontend/appflowy_flutter/pubspec.lock

```diff
@@ -62,8 +62,8 @@ packages:
     dependency: "direct main"
     description:
       path: "packages/appflowy_editor_plugins"
-      ref: "37f35e77"
-      resolved-ref: "37f35e77aefb1b9a0b872086d156b1b885ba02df"
+      ref: aa17c6cb
+      resolved-ref: aa17c6cba53fad811b6e8880dea3a571574557a8
```

Comment:
I have a wild idea that this might be what is causing the generation to fail..
```suggestion
      ref: "aa17c6cb"
      resolved-ref: "aa17c6cba53fad811b6e8880dea3a571574557a8"
```